db/migrate/20150604000000_add_permissions_to_authors.rb

```diff
@@ -0,0 +1,7 @@
+class AddPermissionsToAuthors < ActiveRecord::Migration
+  def change
+    add_column "#{Landable.configuration.database_schema_prefix}landable.authors", :read_access, :boolean, null: :false
+    add_column "#{Landable.configuration.database_schema_prefix}landable.authors", :edit_access, :boolean, null: :false
+    add_column "#{Landable.configuration.database_schema_prefix}landable.authors", :publish_access, :boolean, null: :false
+  end
+end
```

Comment:
hold up... why are we storing this in the db? I thought we agreed that this was going to be front-end only, and there was no reason to store permissions because we're always going to get fresh ones from LDAP on login. Can we talk about this bit?